h2o-core/src/main/java/water/H2O.java

```diff
@@ -116,6 +118,10 @@ public static void printHelp() {
             "    -client\n" +
             "          Launch H2O node in client mode.\n" +
             "\n" +
+            "    -notify <fileSystemPath>" +
```

Comment:
Maybe we could rename it to something like notify_local? Just to make it clear that it is just single node?